lib/routes/ktown4u/artist-brandlist.ts

```diff
@@ -0,0 +1,61 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/artistBrandlist/:grp_no/:grp_no2?',
+    categories: ['shopping'],
+    example: 'ktown4u/artistBrandlist/234590/1723449',
+    parameters: { grp_no: 'artist id (Get in url)', grp_no2: 'product category id (Get in url), empty for all categories' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['cn.ktown4u.com/artistBrandlist?grp_no=:grp_no&grp_no2=:grp_no2'],
```

Comment:
`source` can't match any search parameters.

Reply:
Thanks @TonyRL. Actually, I'm not sure what this does, how would you recommend I modify it, or can I delete it?

Reply:
delete it

Reply:
deleted